frontend/src/components/AdaptableButton.tsx

```diff
@@ -0,0 +1,38 @@
+import dashCircle from '../../images/dashCircle.svg'
+import { useState } from 'react';
+
+interface AdaptableButtonProps {
+    label: string;
+    disabled?: boolean;
+    showDash?: boolean;
+}
+
+const AdaptableButton: React.FC<AdaptableButtonProps> = ({ 
```

Comment:
perhaps we should name this button `CourseButton` just because the current name is very vague